react/getting_started_with_react/rendering_techniques.md

```diff
@@ -301,29 +301,29 @@ function App() {
     </div>
   );
 }
-~~~
+```
 
 Nested ternaries and multiple `&&` operators can be intimidating to look at, so be sure to test things out!
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1.  The React documentation has an excellent guide to [conditional rendering](https://react.dev/learn/conditional-rendering). Strengthen your understanding by reading through it and tackling all of the examples!
-2.  From the same docs, explore more on what you can do with lists by working through the [Rendering Lists article](https://react.dev/learn/rendering-lists). You don't have to worry about the last part on keys, since we will be learning about them in the next lesson.
+1. The React documentation has an excellent guide to [conditional rendering](https://react.dev/learn/conditional-rendering). Strengthen your understanding by reading through it and tackling all of the examples!
+`. From the same docs, explore more on what you can do with lists by working through the [Rendering Lists article](https://react.dev/learn/rendering-lists). You don't have to worry about the last part on keys, since we will be learning about them in the next lesson.
```

Comment:
```suggestion
1. From the same docs, explore more on what you can do with lists by working through the [Rendering Lists article](https://react.dev/learn/rendering-lists). You don't have to worry about the last part on keys, since we will be learning about them in the next lesson.
```